src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -173,7 +173,7 @@ class AdditionalDetailsStep extends React.Component {
      */
     activateWallet(values) {
         const personalDetails = {
-            phoneNumber: LoginUtils.getPhoneNumberWithoutUSCountryCodeAndSpecialChars(values[INPUT_IDS.PHONE_NUMBER]),
+            phoneNumber: parsePhoneNumber(values[INPUT_IDS.PHONE_NUMBER], {regionCode: CONST.COUNTRY.US}).number.significant,
```

Comment:
Why are we assuming country code `US` here? Because gold wallets are US only?

Reply:
I'm not sure, this is not a change introduced by this PR, it was validating only US phones using `getPhoneNumberWithoutUSCountryCodeAndSpecialChars`. Maybe @puneetlath have an idea on this

Reply:
Whoops, sorry I didn't realize this was waiting on me. 

I asked in the #concierge-ops room and Joe and didn't get a clear answer. I think let's leave the behavior as-is here. If there's a reason to change, let's change it separately.